docs/settings.rst

```diff
@@ -215,6 +217,19 @@ The following table describes all available settings.
               execution-environment:
                 container-engine:
 
+  * - container-options
```

Comment:
This hardcoded table is being removed in #564 which will probably be auto-merged within an hour. So there's no need to update this file anymore. You'll need to exclude it from the next rebase.

I don't have any further comments on this PR but I'd like for my other teammates to also look at it first before merging.